### ISSUE TITLE
[Assistive tech and device support] Alert and status messages aren't announced without receiving focus. (11.22.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#46636]
#### VFS team name
MOVE

#### VFS product name
1095B Tax Form

#### VFS feature name
1095B Tax Form - accessibility version

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Alert and status messages aren't announced without receiving focus.

### Issue Details
When a user clicks the download button, they should receive either the [success alert](https://github.com/department-of-veterans-affairs/vets-website/blob/e10f936c067061407844c0a1b8a393edc8944963/src/applications/static-pages/download-1095b/components/App/index.js#L181) or the [error alert](https://github.com/department-of-veterans-affairs/vets-website/blob/e10f936c067061407844c0a1b8a393edc8944963/src/applications/static-pages/download-1095b/components/App/index.js#L153).

When the alert gets inserted into the page, I'm not seeing anything that would cause it to receive focus or to be otherwise announced by a screen reader. **(If I've missed something, please let me know!)**

In the case of the success alert, this isn't critical. The download action by their browser should be announced by screen readers, so the user knows the action was successful because they'll hear that something is being downloaded.

In the case of the error alert, this is a bigger issue. If the alert isn't announced by a screen reader and a file fails to download, then it's not clear to screen reader users that anything has happened at all.

### VA.gov Experience Standard
[Category Number 11, Issue Number 22](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 4.1.3_AA

---

### Platform Recommendation
Two options:

1. When each alert is inserted into the page, set focus to the alert's heading and give the heading a `tabindex="-1"` value. Moving focus to the heading should prompt the screen reader to announce that heading as the next element after pressing the button.
2. For the error alert only, use the [error va-alert component](https://design.va.gov/components/alert#error-alert) instead of the warning alert. The Design System team has included `role="alert"` as part of the error flavor of the alert. Screen readers interpret `role="alert"` as an aria-live region and will immediately announce the contents of the alert without requiring focus to move. This doesn't address the success alert --- but since the success alert only takes place with a browser action that should trigger the screen reader anyway, that's less concerning.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by joshkimux** [comment]
Hi y'all! I'm making updates to our a11y dashboard and noticed a couple of defect-2s were still open here. Checking in real quick to ask if this ticket has been prioritized for fixes or is blocked. 

[We usually recommend fixing `defect-2` issues within 1-2 sprints post release.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-508-defect-2)

If y'all need support, don't hesitate to loop in the CAIA team through the #accessibility-help channel

**Comment by joshkimux** [comment]
Hi @mdewey and @stephte  ! Would love your help on this ticket as part of our[ larger a11y dashboard effort](https://docs.google.com/spreadsheets/d/1lgzMiZZ-0rBPDuB2gRupBAjsyF0WTe005VvEHYrMxww/edit?usp=sharing).
- Is this issue blocked / have any dependencies?
- If not, who would be the best person to fix/resolve this?

Platform recommends to fix `defect-2` issues in 1-2 sprints post launch, but more importantly, this is a huge usability boost for Veterans using screen readers 🙇
